app/preferencesdialog.cpp

```diff
@@ -581,6 +595,11 @@ void FilesPage::autosaveNumberChange(int number)
     mManager->set(SETTING::AUTO_SAVE_NUMBER, number);
 }
 
+void FilesPage::clearRecentFilesList(bool b)
+{
+    mManager->set(SETTING::CLEAR_RECENT_FILES_LIST, b);
+}
+
```

Comment:
Right, that's what I thought too. I'll see if I can come up with a better solution without storing it into a preference key. 

Interesting, haven't experienced that but i'll look into it.